devel/balena-cli/Portfile

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                balena-cli
+version             12.55.2
+revision            0
+categories          devel
+platforms           darwin
+license             Apache-2
+maintainers         { ether.org.za:light @dylanbr } openmaintainer
+
+description         The official balena Command Line Interface.
+long_description    The balena CLI is a Command Line Interface for  \
+                    balenaCloud or openBalena. It builds on the balena API and \
+                    the balena SDK, and can also be directly imported in \
+                    Node.js applications.
+homepage            https://www.balena.io/
+distname            ${name}-v${version}-macOS-x64-standalone
+master_sites        https://github.com/balena-io/${name}/releases/download/v${version}
+
+checksums           rmd160  24a1d9b190fa80c5b65bc6fd941ab28b18f8c6c1 \
+                    sha256  cbef0ef7c46dd81bb410ab1953f1bf130772d1f28361ccfb5f532179e92323da \
+                    size    62587811
+
+use_zip             yes
+use_configure       no
+
+build {}
+
+destroot {
+    xinstall -d -m 755 ${destroot}${prefix}/libexec/${name}
+    copy {*}[glob ${worksrcpath}/*] ${destroot}${prefix}/libexec/${name}
+    ln -s ${prefix}/libexec/${name}/balena ${destroot}${prefix}/bin/balena
+}
```

Comment:
Since we're now installing the pre-built binary, would it make sense to just install the pre-built binary directly from the `${worksrcpath}` directly into `${destroot}${prefix}/bin/`?

Is there a particular reason why we want to keep things in `libexec` and link from there?

Reply:
Actually never mind, I was assuming a single binary for the pre-built.